TidepoolKit/Model/TBloodGlucose.swift

```diff
@@ -37,4 +37,35 @@ public struct TBloodGlucose {
             self.high = high
         }
     }
+
+    public struct StartTarget: Codable, Equatable {
+        public var start: Int?
+        public var target: Double?
+        public var range: Double?
```

Comment:
hm... I find this confusing. The range is a `Double`, but I don't see how a range is made from this value. Should a comment be included?

Reply:
These fields are directly from the Tidepool data model and are documented here: https://developer.tidepool.org/data-model/device-data/types/pumpSettings.html#bgtarget. There are certain combinations of these fields that make sense for certain devices. Specifically, range can be used with either target or high and they mean different things on the different devices. (The joys of trying to standardize diabetes data from many different manufacturers.)

In any case, rather than try to duplicate any documentation from the Tidepool data model, I've chosen to have no documentation here (as it will just grow stale over time) and simply expect that anyone interested will also be looking at the Tidepool data model documentation. Since all of this code "mirrors" the backend API and, thus, the Tidepool data model, I think that is the safest bet. In short, I don't expect anyone to simply take up this code and start uploading data. They will have to understand how this data is used first via the Tidepool data model documentation.